### PR TITLE
Replace Markdown backticks with JavaDoc code tags

### PR DESCRIPTION
**Pull request by wborn**
The proper way to format code with JavaDoc is using code tags and not Markdown backticks.

### CODE REVIEW

**Review by holgerfriedrich — APPROVED** [review]
LGTM

File: bundles/org.openhab.core.io.console.karaf/src/main/java/org/openhab/core/io/console/karaf/internal/CommandWrapper.java [hunk 1]
```diff
@@ -52,8 +52,8 @@ public class CommandWrapper implements Command, Action {
 
     /**
      * The constructor for the "help" instance of this class. This instance will be created by
-     * org.apache.karaf.shell.impl.action.command.ManagerImpl.instantiate(Class<? extends T>, Registry) and
-     * is used to print all usages from the `openhab` scope.
+     * {@code org.apache.karaf.shell.impl.action.command.ManagerImpl.instantiate(Class, Registry)} and
```
Comment (J-N-K) [inline bundles/org.openhab.core.io.console.karaf/src/main/java/org/openhab/core/io/console/karaf/internal/CommandWrapper.java:55]: Is the removal of the type parameter intentional?

**Review by J-N-K — COMMENTED** [review]
Otherwise LGTM

File: bundles/org.openhab.core.io.console.karaf/src/main/java/org/openhab/core/io/console/karaf/internal/CommandWrapper.java (hunk 1 above)
Comment (wborn) [reply bundles/org.openhab.core.io.console.karaf/src/main/java/org/openhab/core/io/console/karaf/internal/CommandWrapper.java]: The removal didn't really improve anything so now it's back!

**Review by J-N-K — APPROVED** [review]

**Pull request merged**